Checksfinder: Fix the last remnant of outdated game description (#4893)
ArchipelagoMW/Archipelago: worlds/checksfinder/docs/en_ChecksFinder.md

## What is considered a location check in ChecksFinder?

> Location checks get submitted when you open all non-bomb cells in a board. The bottom

Reviewer: "get submitted" is an unusual wording. I think cleared, checked, or completed would work better.
```suggestion
Location checks get cleared when you open all non-bomb cells in a board. The bottom
```

> Location check gets submitted when you beat the current board of minesweeper, by opening all non-bomb cells. The bottom
> of the screen has a number next to the archipelago logo, that number is how many location checks is left to be sent with
> your item inventory. You can only get as many checks as you have gained items, plus five to start with being available.

Reviewer 2: ```suggestion
Location checks get submitted when you open all non-bomb cells in a board. The bottom
of the screen has a number next to the Archipelago logo that displays how many location checks are left to be sent with
your current inventory. You can only get as many checks as you have gained items plus five checks to start with.
```
Rewrite for clarity and grammar.
The last sentence wasn't clear to me, so please confirm whether it is accurate.
